src/util/paths.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+const repoRoot = process.cwd(); // expect CLI to be run from consumer repo root
+const appPath = path.resolve(repoRoot, 'src');
+
+module.exports = {
+	repoRoot,
+	appPath,
+};
```

Comment:
I'd like some sort of split but i'm ambivalent about the form it takes. The build config and scripts use a lot of directories and having it all in one place does have a small (perhaps negligible, depending upon your perspective) amount of value.